spec/support/test_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Test
+  class TestController < ActionController::Base
+    use_openstax_exception_rescue
+
+    def bad_action
+      exception = params[:exception].constantize
+      raise case params[:exception]
+            when 'ActionController::RoutingError'
+              exception.new('routing error')
+            when 'Apipie::ParamMissing'
```

Comment:
could just have a made up exception to simulate apipie without the real exception being used? (and so without having to have apipie in the gemspec)